cmd/src/repos_add_kvp.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+	"github.com/sourcegraph/src-cli/internal/api"
+)
+
+func init() {
```

Comment:
IMO, this is a lot of code to put in an `init()`. Style-wise, I think it would be best to move most of this stuff outside the `init()` function and just put the "register the command" step inside. Then it's clear what actually _runs_ on startup.

Reply:
Good point, but I was just following the convention that's used for several other commands I looked at in this repo, and I feel like it's preferable to keep the style consistent